src/routes/(console)/project-[region]-[project]/functions/function-[function]/domains/add-domain/+page.svelte

```diff
@@ -72,7 +72,7 @@
             } else if (behaviour === 'REDIRECT') {
                 rule = await sdk
                     .forProject(page.params.region, page.params.project)
-                    .proxy.createRedirectRule(domainName, $regionalProtocol + redirect, statusCode);
+                    .proxy.createRedirectRule(domainName, redirect, statusCode);
```

Comment:
why is this removed? do we not need the protocol anymore? what changed that its not required anymore?